README.md

```diff
@@ -51,8 +51,8 @@ use cidr_utils::cidr::IpCidr;
 
 let cidr = IpCidr::from_str("192.168.51.0/24").unwrap();
 
-assert_eq!(true, cidr.contains(IpAddr::from_str("192.168.51.103").unwrap()));
-assert_eq!(false, cidr.contains(IpAddr::from_str("192.168.50.103").unwrap()));
+assert_eq!(true, cidr.contains(IpAddr::from([192, 168, 51, 103])));
+assert_eq!(false, cidr.contains(IpAddr::from([192, 168, 50, 103])));
 ```
```

Comment:
Why not use `from_str`?

Reply:
examples should be clean and concise

Reply:
removed this particular change from this PR